ods-setup/setup-jenkins-images.sh

```diff
@@ -66,5 +66,20 @@ ${TAILOR} update ${FORCE} --context-dir=${BASH_SOURCE%/*}/../jenkins/ocp-config
 
 echo "Start Jenkins Builds"
 oc start-build -n ${NAMESPACE} jenkins-master --follow
+STATUS=$(oc get build jenkins-master-1 -o json -n cd | jq '.status.phase' | sed 's/"//g')
```

Comment:
This is easier like this: https://github.com/opendevstack/ods-jenkins-shared-library/blob/master/vars/stageStartOpenshiftBuild.groovy#L33

Reply:
👍